cpp/benchmarks/common/generate_input.cu

```diff
@@ -247,39 +247,43 @@ struct random_value_fn<T, std::enable_if_t<cudf::is_chrono<T>()>> {
  */
 template <typename T>
 struct random_value_fn<T, std::enable_if_t<cudf::is_fixed_point<T>()>> {
-  using rep = typename T::rep;
-  rep const lower_bound;
-  rep const upper_bound;
+  using DeviceType = cudf::device_storage_type_t<T>;
+  using rep        = typename T::rep;
```

Comment:
`cudf::device_storage_type_t<T>` should be exactly the same as `T::rep`, isn't it?

Reply:
Not exactly. It's specialized for decimal32, decimal64, decimal128 alone.
https://github.com/rapidsai/cudf/blob/0fb4d7621de51e58fb63c30c73d35cd01a116ef4/cpp/include/cudf/utilities/type_dispatcher.hpp#L110-L114

 But it should be `T::rep` for decimal alone. For duration and timestamps, it should not be `T::rep`

Reply:
Yeah, this function is only for decimal: `std::enable_if_t<cudf::is_fixed_point<T>()`.
So basically they are the same here.

Reply:
If we can just use one of them, then let's remove the other (`T::rep`) completely.

Reply:
Thank you. That was an oversight. removed `rep` usage.